CMakeLists.txt

```
@@ -192,6 +193,21 @@ if(AMR_WIND_ENABLE_HELICS)
   target_compile_definitions(${amr_wind_lib_name} PRIVATE AMR_WIND_USE_HELICS)
 endif()
 
+if(AMR_WIND_TEST_WITH_PYTHON)
+  set(CMAKE_PREFIX_PATH ${PYTHON_DIR} ${CMAKE_PREFIX_PATH})
+  find_package(Python3 REQUIRED COMPONENTS Interpreter)
+  # Developer note: modify this list if modifying scripts used in python-based regression tests
+  set(PYTHON_REQUIREMENTS numpy netCDF4 matplotlib pathlib pandas)
```

pathlib should be removed since it is core python. Otherwise you get:
`WARNING: Package(s) not found: pathlib` 

```suggestion
  set(PYTHON_REQUIREMENTS numpy netCDF4 matplotlib pandas)
```

---

does that mean I don't need to have import Path from pathlib in the scripts? I think it's in the amrex particle script

---

no you still need to import it, it's just always there as part of python, it's not an external package